tools/vscode-azurewebpubsub/src/tree/hubSettings/EventListeners/EventListenersItem.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { EventListener } from "@azure/arm-webpubsub";
+import { TreeElementBase, createContextValue } from "@microsoft/vscode-azext-utils";
+import { ViewPropertiesModel } from "@microsoft/vscode-azureresources-api";
+import * as vscode from 'vscode';
+import { EventListenerItem } from "./EventListenerItem";
+
+
+export class EventListenersItem implements TreeElementBase {
+    static readonly contextValue: string = 'webPubSubHubEventListenersItem';
+    static readonly contextValueRegExp: RegExp = new RegExp(EventListenersItem.contextValue);
+
+    constructor(public readonly eventListeners: EventListener[]) { }
+
+    async getChildren(): Promise<EventListenerItem[]> {
+        const result: EventListenerItem[] = [];
+        for (let i = 0; i < this.eventListeners.length; i++) {
+            result.push(new EventListenerItem(this.eventListeners[i], i));
+        }
+        return result;
+    }
+
+    getTreeItem(): vscode.TreeItem {
+        return {
+            label: "Event Listeners",
+            iconPath: new vscode.ThemeIcon("list-ordered"),
+            contextValue: createContextValue([EventListenersItem.contextValue]),
+            collapsibleState: this.eventListeners.length > 0 ? vscode.TreeItemCollapsibleState.Expanded : vscode.TreeItemCollapsibleState.None,
+            description: `${this.eventListeners.length} Listeners`
```

Comment:
question: what is the localization process for vscode extensions?

Reply:
I read many extensions source code (spring cloud, container apps, functions, resource group and etc.)
And found all of them actually didn't do the real localization work to support multiple    languages 😆 
All of them wrote `localize()` everywhere but without the rest of work (use some tool to extract the keys of strings, write translations and make it work).
I read the doc of [vscode-nls](https://github.com/Microsoft/vscode-nls) but still don't know how to make it work.

Recommend: keep our extension same as others, only support English, use `localize` to make room for real localization if it's really needed in the future.

Reply:
sounds good to me. I ask because this `description` sounds localizable.

Reply:
now `description` is localized